manifests/00_namespace.yaml

```diff
@@ -9,4 +9,5 @@ metadata:
     workload.openshift.io/allowed: "management"
   labels:
     openshift.io/cluster-monitoring: "true"
+    openshift.io/run-level: "" # specify no run-level turns it off on install and upgrades
```

Comment:
this is touching the wrong NS, per:
https://coreos.slack.com/archives/C03QED56GDS/p1658765567556089?thread_ts=1658523406.064569&cid=C03QED56GDS

Reply:
@sdodson can you make sure we get this right (matches the changes you made to your cluster)

Reply:
Yes, this is the wrong namespace, needs to be `openshift-controller-manager` namespace. Unsure what creates and manages that.

Reply:
yeah, I am changing the wrong namespace, will fix the PR

Reply:
Fixed

Reply:
I'd scoped (rhbz#2110629)[https://bugzilla.redhat.com/show_bug.cgi?id=2110629] at both the operator and operand namespaces, but yes, we only need to fix the operand namespace to unblock 4.11.  Did we want to reduce scope on the bug, or are we ok fixing both namespaces in this pull?